### PR TITLE
feat(ui5-tokenizer): change indicator's text when there are no visible tokens

### PR DESCRIPTION
**Pull request by niyap**
FIXES: #7250 
BGSOFUIRILA-3653

### CODE REVIEW

File: packages/main/test/specs/MultiComboBox.spec.js [hunk 1]
```diff
@@ -1483,6 +1483,31 @@ describe("MultiComboBox general interaction", () => {
...
+			const mcNMore = await $("#mc-more");
+			const tokenizerNItems = await mcNItems.shadow$("ui5-tokenizer");
+			const tokenizerNMore = await mcNMore.shadow$("ui5-tokenizer");
+			const nItemsLabel = await tokenizerNItems.shadow$(".ui5-tokenizer-more-text");
+			const nMoreLabel = await tokenizerNMore.shadow$(".ui5-tokenizer-more-text");
+			let resourceBundleTextItems = null;
+			let resourceBundleTextMore = null;
+			
+	
+			resourceBundleTextItems = await browser.executeAsync(done => {
```
Comment (elenastoyanovaa) [inline packages/main/test/specs/MultiComboBox.spec.js:1498]: you can concat this in one browser.executeAsync, just pass a {} as a result of done.

File: packages/main/test/specs/MultiInput.spec.js [hunk 2]
```diff
@@ -199,6 +199,31 @@ describe("MultiInput general interaction", () => {
...
+		const tokenizerNItems = await miNItems.shadow$("ui5-tokenizer");
+		const tokenizerNMore = await miNMore.shadow$("ui5-tokenizer");
+		const nItemsLabel = await tokenizerNItems.shadow$(".ui5-tokenizer-more-text");
+		const nMoreLabel = await tokenizerNMore.shadow$(".ui5-tokenizer-more-text");
+		let resourceBundleTextItems = null;
+		let resourceBundleTextMore = null;
+		
+
+		resourceBundleTextItems = await browser.executeAsync(done => {
+			const mi = document.getElementById("mi-items");
```
Comment (elenastoyanovaa) [inline packages/main/test/specs/MultiInput.spec.js:215]: same

**Review by elenastoyanovaa — CHANGES_REQUESTED** [review]
is it applicable to add tests for the standalone tokenizer?

**Comment by niyap** [comment]
> is it applicable to add tests for the standalone tokenizer?

Currently, there is no separate file for ui5-tokenizer related tests. We can plan such action in future

**Review by elenastoyanovaa — CHANGES_REQUESTED** [review]
Please add an issue link or JIRA in the PR description

**Review by elenastoyanovaa — APPROVED** [review]

**Pull request merged**